typings/index.d.ts

```diff
@@ -2422,7 +2424,9 @@ declare module 'discord.js' {
 
   interface ChannelPosition {
     channel: ChannelResolvable;
-    position: number;
+    position?: number;
+    parent?: ChannelResolvable;
+    lockPermissions?: boolean;
```

Comment:
Properties should be sorted alphabetically:
```suggestion
    lockPermissions?: boolean;
    parent?: ChannelResolvable;
    position?: number;
```